base_layer/core/src/chain_storage/lmdb_db/lmdb_db.rs

```diff
@@ -1366,6 +1389,14 @@ impl LMDBDatabase {
                 "utxo_commitment_index",
             )?;
         }
+        for commitment in burned_outputs {
```

Comment:
perf: Is there a reason that you have to loop through burned commitments separately? You should be able to delete from the commitment index on line 1317.

Reply:
They only get added in ln 1374, I can't add it before that. 
I made the call here to rather than optimise and create a unique path into the database, I reuse the existing tested paths for their few outputs. 

So yes technically I add, then remove them, but by doing it this way, I ensure they get stored in all the correct spent db's with the correct keys.

Reply:
Thanks looked again and it makes sense